src/OrbitDB.js

```diff
@@ -260,6 +274,23 @@ class OrbitDB {
       await this._pubsub.unsubscribe(address)
     }
 
+    // Cache cleanup
+    // First remove the handler by db address
+    // Then close the cache if there are no more handlers
+    for(let directory of Object.keys(this.caches)) {
+      const cache = this.caches[directory] || this.cache
+      const store = this.stores[address]
+
+      if (store && store.options.directory == directory) {
```

Comment:
Maybe a typo but think we should stick with triple === throughout the codebase, any reason not to?